client/src/pages/SignUp.js

```diff
@@ -0,0 +1,238 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Typography,
+  TextField,
+  Card,
+  CardContent,
+  Grid,
+} from "@material-ui/core";
+import { Link } from "react-router-dom";
+import { withStyles } from "@material-ui/core/styles";
+import BackgroundImg from "../Assets/images/signUpBkg.png";
+import ChefsMenuLogo from "../Assets/images/Logo.png";
+
+const regPattern = new RegExp(/^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/i);
+
+const signUpPageStyle = (theme) => ({
+  signUpContainer: {
+    fontFamily: '"Montserrat"',    
```

Comment:
```suggestion
    fontFamily: '"Roboto"',    
```
As per our conversation in standup yesterday